ts-tests/integration-tests/evm-contract.test.ts

```diff
@@ -147,7 +148,7 @@ describeLitentry('Test EVM Module Contract', ``, (context) => {
             return data;
         };
 
-        const message = await sayMessage(deployed.contractAddress!);
+        const message = (await sayMessage(deployed.contractAddress!)) as unknown as string;
```

Comment:
If we know that message should always be `string`, how about making the return type of `sayMessage` explicit:
 `const sayMessage = async (contractAddress: string): Promise<string> => {...}`